modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -5583,6 +5582,11 @@ public ObjectActionModel newObjectActionModel(
 					fragmentEntryLinkModel.getFragmentEntryLinkId()));
 		}
 
+		layoutPageTemplateStructureRelModel.setSegmentsExperienceId(
+			fragmentEntryLinkModels.get(
```

Comment:
@slnn Please don't chain it here.

Reply:
Oh, sorry, SF does not complain it, so I use it

Reply:
Forced push to apply the change